src/lib/schematics/ng-update/test-cases/v8/material-subpackage_expected_output.ts

```diff
@@ -0,0 +1,70 @@
+import {NgModule} from '@angular/core';
+import {
+  MatAutocompleteModule,
+  MatBadgeModule,
+  MatButtonModule,
+  MatButtonToggleModule,
+  MatCardModule,
+  MatCheckboxModule,
+  MatChipsModule,
+  MatDatepickerModule,
+  MatDialogModule,
+  MatExpansionModule,
+  MatIconModule,
+  MatInputModule,
+  MatListModule,
+  MatMenuModule,
+  MatNativeDateModule,
+  MatPaginatorModule,
+  MatProgressBarModule,
+  MatProgressSpinnerModule,
+  MatRadioModule,
+  MatRippleModule,
+  MatSelectModule,
+  MatSidenavModule,
+  MatSlideToggleModule,
+  MatSnackBarModule,
+  MatSortModule,
+  MatTableModule,
+  MatTabsModule,
+  MatToolbarModule,
+  MatTooltipModule
+} from '@angular/material';
```

Comment:
Is that test-case incorrect? Shouldn't it import from the secondary entry-points?

Reply:
I think you might have not added the rule to he upgrade rules right?

https://github.com/angular/material2/blob/f82259bac2169a049977b4b64cab3a6e52b79bb1/src/lib/schematics/ng-update/index.ts#L16-L27

Reply:
Your right, it looks like it is still not executing.  I am not sure what I am missing to have it included as it should be matched by the glob in the BAZEL file.

Reply:
Yeah the Bazel glob should only transpile that file. We still need to add that rule to the `ng-update/index.ts` file, so that TSLint can pick it up.